src/sentry/db/models/manager.py

```diff
@@ -323,7 +323,7 @@ def from_event_id(self, id_or_event_id, project_id):
         event_id = normalize_event_id(id_or_event_id)
         if not event_id:
             logger.warning('Attempt to fetch SnubaEvent by primary key', exc_info=True, extra={
-                'stack': True
```

Comment:
was this actually doing anything ?

Reply:
Yea, it was send along the key value pair "stack": True every time :)

Reply:
are you sure it was not  required by the FORMAT string ?
https://docs.python.org/2/library/logging.html#logging.Logger.debug

Reply:
Yeah, `extra` is only for user defined attributes